nas/README.md

```diff
@@ -26,12 +26,15 @@ which may conflict with each other.
 
 ## Baseline methods implemented
 
-Description of each baseline method.
+We have implemented four NAS baseline methods:
+1. random search: we explore the search space via morphing the network architectures randomly, so the actual performance of the generated nerual architecture has no effect on later search.
+2. grid search: we manually specified subset of the hyperparameter space to search, i.e., the number of layers and the width of the layers are predefined.
+3. greed search: we explore the search space in a greedy way. The "greedy" here means the base architecture for the next iteration of search is chosen from those generated by current iteration, the one that have best performance on the training/validation set in our implementation.
+4. bayesian optimization: it's the default search strategy of Auto-Keras currently. refer [arXiv:1806.10282](https://arxiv.org/abs/1806.10282) for more detail. 
 
 ## How to run the baseline methods?
 
-Code example containing two parts: data preparation, search.
-Should call CnnModule.
+Refer to examples/nas/cifar10_tutorial.py for more details.
 
 
 ## How to implement your own search?
```

Comment:
We need content of this part, too.
Let them know which function to override, and what are the meanings of the parameters.